testsuite/scripts/samples/test_electrophoresis.py

```diff
@@ -20,29 +20,29 @@
 import numpy as np
 
 sample, skipIfMissingFeatures = importlib_wrapper.configure_and_import(
-    "@SAMPLES_DIR@/electrophoresis.py")
+    "@SAMPLES_DIR@/electrophoresis.py", N_SAMPLES=4000)
 
 
 @skipIfMissingFeatures
 class Sample(ut.TestCase):
     system = sample.system
 
-    @ut.skipIf(tuple(map(int, np.__version__.split("."))) < (1, 10),
-               "not supported for numpy < v1.10")
     def test_persistence_length(self):
-        value = np.mean(sample.fit[0])
-        self.assertTrue(5. < value < 110., "length = {:.0f}".format(value))
+        self.assertTrue(27. < sample.opt[0] < 30.,
+                        "length = {:.1f}".format(sample.opt[0]))
+        self.assertTrue(
+            32. < sample.opt_obs[0] < 36., "length = {:.1f}".format(sample.opt_obs[0]))
 
     def test_mobility(self):
-        value = sample.mu
-        self.assertLess(abs(value), 0.45, "mobility = {:.2f}".format(value))
+        self.assertTrue(abs(sample.mu - 10.2) < 0.1,
+                        "mobility = {:.2f}".format(sample.mu))
```

Comment:
The error message
```
  File "testsuite/scripts/samples/test_electrophoresis.py", line 38, in test_mobility
    "mobility = {:.2f}".format(sample.mu))
AssertionError: False is not true : mobility = 10.26
```
is not very helpful, you may consider instead:
```python
self.assertAlmostEqual(sample.mu, 10.2, delta=0.1)
```
which outputs
```
  File "testsuite/scripts/samples/test_electrophoresis.py", line 37, in test_mobility
    self.assertAlmostEqual(sample.mu, 10.2, delta=0.1)
AssertionError: 10.341812051847997 != 10.2 within 0.1 delta
```